tests/contrib/celery/test_integration.py

```diff
@@ -650,7 +649,7 @@ def fn_task_parameters(user, force_logout=False):
         assert len(traces) == 2
         assert len(traces[0]) + len(traces[1]) == 3
 
-    @flaky(1742580778)
+    @flaky(1746305579, reason="Test returns an unfixed number of celery runs so it never passes the last assertion")
```

Comment:
The way that assertion is written is intended to account for this, so I'm surprised it's failing.

Reply:
Resolving this comment since for some reason, it passes now when I remove the flaky decorator and I don't want to question the beat scheduler :)